lib/location/scan_wifi.c

```diff
@@ -62,9 +88,10 @@ void scan_wifi_details_get(struct location_data_details *details)
 #if defined(CONFIG_LOCATION_METHOD_WIFI_NET_IF_UPDOWN)
 static int scan_wifi_interface_shutdown(struct net_if *iface)
 {
-	LOG_DBG("Shutting down Wi-Fi interface");
 	int ret;
 
+	LOG_DBG("Shutting down Wi-Fi interface");
```

Comment:
@simensrostad had some theories about shut down not happening properly. Is that theory thrown away? We don't need to wait anything to shut down properly but `net_if_down` handles it completely.